Don't come close to quilt or other package-related files.
Fixes problem with removing 0-sized files from .pc dir.
--- a/Makefile
+++ b/Makefile
@@ -163,7 +163,7 @@ FORCE: ;
 ###
 # clean: remove generated files
 # mrproper does a full cleaning including .config and linux symlink
-FIND_IGNORE := \( -name .git \) -prune -o
+FIND_IGNORE := \( -name .git -o -path ./.pc -o -path ./debian \) -prune -o
 quiet_cmd_rmfiles = $(if $(wildcard $(rm-files)),RM     $(wildcard $(rm-files)))
       cmd_rmfiles = rm -f $(rm-files)
 clean: srctree
